packages/next/src/executors/build/build.impl.ts

```diff
@@ -71,7 +71,7 @@ export default async function buildExecutor(
   }
 
   if (!directoryExists(options.outputPath)) {
-    mkdir(options.outputPath);
+    mkdirSync(options.outputPath);
```

Comment:
```suggestion
    await mkdir(options.outputPath, { recursive; true });
```

This would best mimic `mkdir` from `fs-extra` as it always passes `recursive: true` to the underlying `node:fs/promises` API.

Reply:
Makes sense! 👍